.github/ISSUE_TEMPLATE/mentoring_request.md

```diff
@@ -15,3 +15,7 @@ Answer: **I have previous experience in React.**
 **Is there any particular issue you'd like to work on? You may want to check out the [list of good first issues](https://github.com/argoproj/argo-workflows/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).**
 
 Answer: **I would like to work with issue #7920**
+
+**Do you want to work with your mentor async (i.e. by Slack messaging or issue comments), or sync (e.g. using Zoom video)?** Please include your time-zone if you want to work sync.
```

Comment:
Do we need Slack usernames for each mentor?

Reply:
not sure, I've not needed it yet